airflow/www/static/js/tree/renderTaskRows.jsx

```diff
@@ -160,7 +172,7 @@ const Row = (props) => {
           </Collapse>
         </Td>
         <Td width={0} p={0} borderBottom={0} />
-        <Td p={0} align="right" _groupHover={{ backgroundColor: 'rgba(113, 128, 150, 0.1)' }} transition="background-color 0.2s" borderBottom={0}>
+        <Td p={0} align="right" _groupHover={{ backgroundColor: 'rgba(113, 128, 150, 0.1)' }} transition="background-color 0.2s" borderBottom={0} width={`${dagRunIds.length * 16}px`}>
```

Comment:
Where does the 16px per dagRun come from? Is there anyway to not have this hardcoded here but somehow worked out based on what the Column tells us?

Secondly we have this in `16px` two places.

Thirdly: on L111 we have 18px wide box for empty TI -- shouldn't that be the same width? (Not 100% sure)

Reply:
Yes, I just updated statusbox and taskrows to share a single value + padding for the box width and overall column width